<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerDistributionDialogNew</class>
 <widget class="QDialog" name="TrackerDistributionDialogNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>283</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Distribution Function Parameters</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNpart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of particles</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Longitudinal Distribution</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rmin [0, Rmax-0.1]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxRmin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rmax [0,6]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxRmax">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLGaussian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLUniform">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniform</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>495</width>
     <height>148</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Beta_x1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxBx1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Beta_y1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxBy1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>eta_1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEta1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Alpha_x1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxAx1">
      <property name="minimum">
       <double>-10000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Alpha_y1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxAy1"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>eta_1p</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEta1p"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Beta_x2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxBx2"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Beta_y2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxBy2"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>eta_2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEta2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Alpha_x2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxAx2"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Alpha_y2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxAy2"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>eta_2p</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEta2p"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>413</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxU">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>eps_1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEps1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>eps_2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxEps2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>477</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>alpha_L</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxAlpha_L">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>sigma_s</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxSigma_s">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>sigma_p</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScientificDoubleSpinBox" name="doubleSpinBoxSigma_p">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="deltaFunctionCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delta Function Distribution</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxTransverse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Transverse Distribution </string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxRmin_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cutoff [sigma]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTGaussian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTUniform">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniform</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ScientificDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrackerDistributionDialogNew</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrackerDistributionDialogNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
